Emby.Server.Implementations/ScheduledTasks/Tasks/DeleteTranscodeFileTask.cs

```diff
@@ -39,7 +39,13 @@ public DeleteTranscodeFileTask(ServerApplicationPaths appPaths, ILogger logger,
         /// Creates the triggers that define when the task will run
         /// </summary>
         /// <returns>IEnumerable{BaseTaskTrigger}.</returns>
-        public IEnumerable<TaskTriggerInfo> GetDefaultTriggers() => new List<TaskTriggerInfo>();
+        public IEnumerable<TaskTriggerInfo> GetDefaultTriggers()
+        {
+            yield return new TaskTriggerInfo
+            {
+                Type = TaskTriggerInfo.TriggerInterval, IntervalTicks = TimeSpan.FromHours(24).Ticks
```

Comment:
tut tut, this doesn't feel right, you're sneakily enabling a cleanup task which we decided to keep off by default ;)

Reply:
I wasn't aware that was a deliberate choice, I'll revert that one.

Reply:
I don't think there's any harm in enabling this though, especially since the files will have no use after more than two or three hours if they're from an especially long movie or something.

Reply:
They have their use if someone paused a movie and then went away for a month... right @sparky8251 ?

Reply:
Alright, I removed this which means the only actual changes this PR has are standardizing the variable name and warning fixes.